python/Ganga/GPIDev/Credentials/GridProxy.py

```diff
@@ -206,12 +204,10 @@ def chooseCommandSet(self):
         Return value: None
         """
         from Ganga.GPIDev.Lib.GangaList.GangaList import GangaList
-        if "ICommandSet" == self.command._name or isType(self.command, GangaList):
-            if self.voms:
-                self.command = self.vomsCommand
-            else:
-                self.command = self.gridCommand
-        return None
+        if self.voms:
```

Comment:
That would be technically possible but this PR is also fixing other problems in that it's currently impossible to change VO at runtime. This PR allows a script to set which VO it is working with which is useful even outside of the tests and makes the function `chooseCommandSet` actually do what it is supposed to.

There'll be no need for the tests to do any wgetting, we will simply provide the config file in a known location.